app/controllers/quote_anonymous_controller.rb

```diff
@@ -0,0 +1,18 @@
+# Entry point for the quotes_anonymous resource
+class QuoteAnonymousController < ApplicationController
+  before_action :ilid?
+
+  def update
+    quote = Quote.find_by_id!(params[:id])
+
+    unless quote.can_be_anonymized_by?(current_user)
+      flash.now[:error] = 'Je kan alleen je eigen quote anoniem maken'
+      return render turbo_stream: turbo_stream.update('flash', partial: 'layouts/alert')
+    end
+
+    quote.user_id = nil
+    quote.reporter_id = nil
+    quote.save!
```

Comment:
Dit zijn wel veel stappen voor in een controller, liever dit naar de Quote model delegeren. 

Method maken in de quote model met deze stappen en dan hier alleen `quote.anonymize!` callen.